src/app/components/ErrorFormatter/index.tsx

```diff
@@ -27,20 +28,34 @@ export function ErrorFormatter(props: Props) {
     [WalletErrors.InvalidNonce]: t('errors.invalidNonce'),
     [WalletErrors.DuplicateTransaction]: t('errors.duplicateTransaction'),
     [WalletErrors.NoOpenWallet]: t('errors.noOpenWallet'),
-    [WalletErrors.USBTransportError]: t('errors.usbTransportError', 'USB Transport error : {{message}}', {
-      message,
-    }),
+    [WalletErrors.USBTransportError]: t(
+      'errors.usbTransportError',
+      'USB Transport error : {{message}}. This usually means your browser is not supported - Chrome should work, but Firefox does not support WebUSB.',
```

Comment:
The space on both the left and right side of : is some French specialty.

```suggestion
      'USB Transport error: {{message}}. This usually means your browser does not support WebUSB. Try using a Google Chrome-based browser and check application permissions.',
```

Reply:
I disagree. This is more accurate for like.. a linux user. But no one else will understand.

Reply:
cc @tjanez

Reply:
'USB Transport error: {{message}}. This usually means your browser does not support WebUSB (e.g. Firefox). Try using Chrome and check application permissions.'